src/core/virtual_sites/lb_inertialess_tracers.cpp

```diff
@@ -157,17 +150,20 @@ void CoupleIBMParticleToFluid(Particle &p) {
  *  The fluid velocity is obtained by linear interpolation,
  *  cf. eq. (11) in @cite ahlrichs99a.
  */
-void GetIBMInterpolatedVelocity(const Utils::Vector3d &pos, Utils::Vector3d &v,
-                                Utils::Vector3d &forceAdded) {
+template <bool ReturnVelocity>
+Utils::Vector3d GetIBMInterpolatedVelocity(Utils::Vector3d const &pos) {
```

Comment:
It is out of the scope for this PR but this function clearly is in a really bad state...